src/views/preview/presentation.jsx

```diff
@@ -131,22 +132,22 @@ const PreviewPresentation = ({
     const showNotesAndCredits = editable || projectInfo.description ||
         (!projectInfo.instructions && !projectInfo.description); // show if both are empty
 
-    // Allow embedding html in banner messages coming from the server
-    const embedCensorMessage = message => (
-        // eslint-disable-next-line react/no-danger
-        <span dangerouslySetInnerHTML={{__html: message}} />
-    );
-
     let banner;
     if (visibilityInfo.deleted) { // If both censored and deleted, prioritize deleted banner
         banner = (<Banner
             className="banner-danger"
             message={<FormattedMessage id="project.deletedBanner" />}
         />);
     } else if (visibilityInfo.censored) {
+        const censoredMessage = (
+            <CensoredMessage
+                messageHTML={visibilityInfo.messageHTML}
```

Comment:
The visibilityInfo has a property called `message`, but it doesn't have one called `messageHTML`